sdk/containerregistry/azure-containers-containerregistry/src/test/resources/session-records/ContainerRegistryBlobClientIntegrationTests.downloadManifest[1].json

```diff
@@ -57,46 +57,46 @@
       "retry-after" : "0",
       "Range" : "0-0",
       "StatusCode" : "202",
-      "Date" : "Thu, 26 Jan 2023 06:12:17 GMT",
+      "Date" : "Sun, 12 Feb 2023 21:27:32 GMT",
       "Docker-Distribution-Api-Version" : "registry/2.0",
-      "X-Ms-Correlation-Request-Id" : "c628379e-43c7-4c2b-a3c5-bf7a92d33217",
+      "X-Ms-Correlation-Request-Id" : "11cea6ac-e49d-48e0-a921-5d7df603602d",
       "Access-Control-Expose-Headers" : "Docker-Content-Digest,WWW-Authenticate,Link,X-Ms-Correlation-Request-Id",
       "Strict-Transport-Security" : "max-age=31536000; includeSubDomains,max-age=31536000; includeSubDomains",
-      "X-Ms-Client-Request-Id" : "ac970cfa-f247-4a48-87a0-dccbca14c1a6",
-      "X-Ms-Request-Id" : "36ae8999-6352-45e0-9d6a-93fa88c1620a",
-      "Docker-Upload-Uuid" : "52eae528-ce53-4ae9-b27a-96e13fed221a",
+      "X-Ms-Client-Request-Id" : "e2e99c65-1cc7-4ec6-b76a-6ed2e3ab683c",
+      "X-Ms-Request-Id" : "e2eda780-d5af-4a1c-8a23-eb3236337952",
+      "Docker-Upload-Uuid" : "f6865a0e-8855-400e-a4a2-7e64e1a93a20",
```

Comment:
Does the UUID need to be redacted?

Reply:
it should work without redaction as long as it's consistent within this file.